test/unit/publicFolderTest.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const path = require('path')
+const generateTestApp = require('../util/generateTestApp')
+const cleanupTestApp = require('../util/cleanupTestApp')
+const fork = require('child_process').fork
+const fse = require('fs-extra')
+const klawSync = require('klaw-sync')
+const request = require('supertest')
+
+describe('Public folder section tests', function () {
+  // path to the directory where the test app is located
+  const appDir = path.join(__dirname, '../', 'app', 'publicFolderTest')
+  // options to pass into generateTestApp
+  let options = {rooseveltPath: '../../../roosevelt', method: 'startServer'}
+
+  // package.json source code
+  let packageSource = `{ "version": "0.5.1", "rooseveltConfig": {}}`
+
+  beforeEach(function (done) {
+    // start by copying the alreadly made mvc directory into the app directory
+    fse.copySync(path.join(__dirname, '../', 'util', 'mvc'), path.join(appDir, 'mvc'))
+    done()
+  })
+
+  afterEach(function (done) {
+    cleanupTestApp(appDir, (err) => {
+      if (err) {
+        throw err
+      } else {
+        done()
+      }
+    })
+  })
+
+  it('should be able to send a web page back with no problem with favicon set to something', function (done) {
```

Comment:
In this test could you look for a way to prove that the favicon is loaded with the page?

Reply:
Also, once you have that figured out add 2 more tests for maximum coverage:
- Have favicon set  to null and test that no favicon is loaded
- Have the favicon set to the wrong path where there is no image and test that no favicon is loaded